src/config.rs

```diff
@@ -635,7 +642,7 @@ impl Default for DefaultCfConfig {
             max_bytes_for_level_base: ReadableSize::mb(512),
             target_file_size_base: ReadableSize::mb(8),
             level0_file_num_compaction_trigger: 4,
-            level0_slowdown_writes_trigger: None,
+            level0_slowdown_writes_trigger: Some(20),
```

Comment:
How are those default value numbers determined? Do we have such a document? I am afraid simply change the default value could bring some unexpected behavior when customers upgrades to newer version.

Reply:
The default value for this config parameter is 20 which is written in the TiKV config documentation (which is the default value set by rocksdb). I just set it temporarily, and I will configure a better way for it.